apps/test-providers/src/ui/content/SampleContentControl.scss

```diff
@@ -9,4 +9,5 @@
   width: 100%;
   justify-content: center;
   user-select: none;
+  -webkit-user-select: none;
```

Comment:
You might want to add a build step that automatically adds vendor prefixes (in the future).